### PR TITLE
Added Desktop version

### PR DESCRIPTION
**Pull request by Dredwell**
Added Desktop version to my portfolio using media queries. I created a new CSS file where are all desktop styles.

I fixed some issues in the mobile version

### CODE REVIEW

File: styles/desktop-styles.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+footer .social-media {
+  margin: 18px 0;
+}
+
+.myemail,
+.indicator2 {
+  display: none;
+}
```
Comment (mmsesay) [inline styles/desktop-styles.css]: - [ ] You did well working on this task. However, creating a new CSS file for the desktop version is not a good practice. As per this project requirement, you are required to use media queries to handle the responsiveness for the desktop version. Please take a look at [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) to get more understanding about implementing media queries.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Dredwell,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response. 😄

**Review by mke2111 — COMMENTED** [review]
Hello @Dredwell & @GrebilCruz  👋,

_This is an additional review as a result of the second opinion request._

The media queries work just fine in the HTML file, kudos to you. 👍🏽 

Kindly note that as requested by the previous reviewer, the [ project guidelines](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) require you to add and define the media queries inside the CSS file and not in the HTML file. 

Screenshot 2022-03-31 202514

Use the CSS format below
```
@media screen and (min-width: 1024px) {
/*
.
.
*/
}

```

Keep rocking �
�!

File: styles/styles.css [hunk 2]
```diff
@@ -356,3 +369,159 @@ footer {
...
+  }
+
+  header {
+    background-image: url(../media/Header-llustration-desktop.png);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-position: 0;
+    background-size: cover;
+  }
```
Comment (ShadyShawkat) [inline styles/styles.css:422]: - [ ] Kindly make sure that your design looks fine and matches the Figma design on width `768px`, [as per requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=(Desktop%3A%20768px%20or%20wider).).

- On width `900px` to `768px`, The project cards don't get any smaller and they start taking more width than they should, which leads to the following behavior.
chrome-capture-2022-2-31
image

[Hint] You can easily fix this by using `minmax` for the `grid-template-columns` property. 
https://github.com/Dredwell/my-portfolio/blob/1cabc16c7d32cf13d43eae70b6303ea87fe5f248/styles/styles.css#L445
You can read more about this [here](https://css-tricks.com/you-want-minmax10px-1fr-not-1fr/).
After doing so, kindly fix the display of technologies for each project card on width `768px`.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Dredwell, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Dredwell** [comment]
@mke2111 Thanks for the feedback.


https://user-images.githubusercontent.com/85886248/161269034-5636026f-5594-4734-acdb-0ae300a39c5b.mp4


But, as you can see, the cards have the desired behavior. The cards have the same width as the Figma design (367px).

**Comment by Dredwell** [comment]
@ShadyShawkat Thanks for the Feedback

https://user-images.githubusercontent.com/85886248/161291241-7f094f66-03ed-46aa-b514-9d29746fad6e.mp4

As you can see, my cards have the desired behavior and then don't need any change. I don't know why you have different behavior on my website than me. They have the same width as the Figma template.

**Review by Davidosky007 — COMMENTED** [review]
Hello @Dredwell  👋,

_This is an additional review as a result of the second opinion request._

Nice job 👏 working on the right media query for your desktop designs the break point ( 768 px - upward) works properly and displays the right responsive design  for desktop device while ( 767 px - downward) displays the mobile version as it should since the mobile first approach is being used 💯 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by OLIPLICHE — APPROVED** [review]
Hi @Dredwell, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**